tests/InfobipMessageTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace NotificationChannels\Infobip\Tests;
+
+use NotificationChannels\Infobip\InfobipMessage;
+use NotificationChannels\Infobip\Test\TestCase;
+
+class InfobipMessageTest extends TestCase
+{
+    /** @test */
+    public function it_can_accept_a_content_when_constructing_a_message(): void
```

Comment:
Could you add some more tests to cover the channel please?

Reply:
Doesn't seem to be any tests here still.

Reply:
Am stuck on this, I cant seem to be able to write any test on the channels, can you assist?

Reply:
Please assist, I really need to use this package for production and am drawing a blank on the channel test